docs/modules/publisher-commandline.md

```diff
@@ -0,0 +1,383 @@
+[Home](../../readme.md)
+
+### OPC Publisher Command Line Arguments for Version 2.5 and below
```

Comment:
>for Version 2.5 and below [](http://example.com/codeflow?start=41&length=25)

I think the order of the topics could be confusing for the user. This repo does not contain 2.5 or below code/versions. It would make more sense to start with 2.8 command line arguments and add "2.5 and below" arguments at the bottom of the doc.

Reply:
changed